prow/config.yaml

```diff
@@ -556,83 +556,15 @@ presubmits:
     spec:
       <<: *common_e2e_spec
 
-  - name: daily-e2e-rbac-no_auth
+  - name: daily-e2e
     agent: kubernetes
-    context: prow/daily-e2e-rbac-no_auth.sh
+    context: prow/daily-e2e.sh
     always_run: true
-    rerun_command: "/test daily-e2e-rbac-no_auth"
-    trigger: "(?m)^/(retest|test (all|daily-e2e-rbac-no_auth))?,?(\\s+|$)"
-    branches:
-    - master
-    labels:
-      preset-service-account: true
-    spec:
-      containers:
-      - image: gcr.io/istio-testing/prowbazel:0.4.8
-        args:
-        - "--repo=github.com/istio-releases/daily-release=$(PULL_REFS)"
-        - "--clean"
-        - "--timeout=60"
-        # Bazel needs privileged mode in order to sandbox builds.
-        securityContext:
-          privileged: true
-      nodeSelector:
-        cloud.google.com/gke-nodepool: new-cluster-pool
-    run_after_success:
-    - name: daily-e2e-rbac-no_auth-skew
-      agent: kubernetes
-      context: prow/daily-e2e-rbac-no_auth-skew.sh
-      always_run: true
-      rerun_command: "/test daily-e2e-rbac-no_auth-skew"
-      trigger: "(?m)^/(retest|test (all|daily-e2e-rbac-no_auth-skew))?,?(\\s+|$)"
-      branches:
-      - master
-      labels:
-        preset-service-account: true
-      spec:
-        containers:
-        - image: gcr.io/istio-testing/prowbazel:0.4.8
-          args:
-          - "--repo=github.com/istio-releases/daily-release=$(PULL_REFS)"
-          - "--clean"
-          - "--timeout=120"
-          # Bazel needs privileged mode in order to sandbox builds.
-          securityContext:
-            privileged: true
-        nodeSelector:
-          cloud.google.com/gke-nodepool: new-cluster-pool
-
-  - name: daily-e2e-rbac-no_auth-default
-    agent: kubernetes
-    context: prow/daily-e2e-rbac-no_auth-default.sh
-    always_run: true
-    rerun_command: "/test daily-e2e-rbac-no_auth-default"
-    trigger: "(?m)^/(retest|test (all|daily-e2e-rbac-no_auth-default))?,?(\\s+|$)"
-    branches:
-    - master
-    labels:
-      preset-service-account: true
-    spec:
-      containers:
-      - image: gcr.io/istio-testing/prowbazel:0.4.8
-        args:
-        - "--repo=github.com/istio-releases/daily-release=$(PULL_REFS)"
-        - "--clean"
-        - "--timeout=60"
-        # Bazel needs privileged mode in order to sandbox builds.
-        securityContext:
-          privileged: true
-      nodeSelector:
-        cloud.google.com/gke-nodepool: new-cluster-pool
-
-  - name: daily-e2e-rbac-auth
-    agent: kubernetes
-    context: prow/daily-e2e-rbac-auth.sh
-    always_run: true
-    rerun_command: "/test daily-e2e-rbac-auth"
-    trigger: "(?m)^/(retest|test (all|daily-e2e-rbac-auth))?,?(\\s+|$)"
+    rerun_command: "/test daily-e2e"
+    trigger: "(?m)^/(retest|test (all|daily-e2e))?,?(\\s+|$)"
     branches:
     - master
+    - release-0.6
```

Comment:
why is this required?
what happens when we run for  branch release-0.8 here

Reply:
I'm unsure if or why they are needed. That line was present in the configurations for the daily-e2e-cluster_wide tests and I think I miscommunicated with Seb, thinking we should keep them.